core/src/main/scala/cats/data/OneAnd.scala

```diff
@@ -104,7 +111,7 @@ private[data] sealed trait OneAndInstances extends OneAndLowPriority2 {
     }
 
   implicit def oneAndSemigroup[F[_]: MonadCombine, A]: Semigroup[OneAnd[F, A]] =
-    oneAndSemigroupK.algebra
+    oneAndSemigroupK(MonadCombine[F]).algebra
```

Comment:
for some reason the compiler get confused with `oneAndSemigroupK[F]` after adding `import cats.std.list._` on the top of the file. It says that there is an ambiguity between `MonadCombine[List]` and `MonadCombine[F]` ...